src/praisonai/praisonai/bots/telegram.py

```
@@ -720,4 +772,51 @@ def _convert_update_to_message(self, update, override_text: Optional[str] = None
+            keyboard = PairingUIBuilder.create_telegram_keyboard(
+                user_name=user_name,
+                code=code, 
+                channel=channel,
+                user_id=user_id
+            )
+            
+            message = await self._application.bot.send_message(
+                chat_id=owner_user_id,
+                text=f"*{user_name}* wants to chat. Approve access?",
+                parse_mode="Markdown",
+                reply_markup=InlineKeyboardMarkup.from_dict(keyboard)
```

Ruff (0.15.10)

[error] Undefined name `InlineKeyboardMarkup`

(F821)